docs/java-rest/high-level/apis/bulk.asciidoc

```diff
@@ -0,0 +1,103 @@
+[[java-rest-high-document-bulk]]
+=== Bulk API
+
+[[java-rest-high-document-bulk-request]]
+==== Bulk Request
+
+A `BulkRequest` can be used to execute multiple index, update and/or delete
+operations using a single request.
+
+It requires at least one operation to be added to the Bulk request:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/CRUDDocumentationIT.java[bulk-request]
+--------------------------------------------------
+<1> Creates the Bulk request
+<2> Adds an `IndexRequest` to the Bulk request. See <<java-rest-high-document-index>>
+for more information on how to build `IndexRequest`.
+<3> Adds a `UpdateRequest` to the Bulk request.
+<4> Adds a `DeleteRequest` to the Bulk request. See <<java-rest-high-document-delete>>
+for more information on how to build `DeleteRequest`.
+
+WARNING: The Bulk API supports only documents encoded in JSON. Providing documents in
+any other format will result in an error.
```

Comment:
doesn't it also support SMILE? Or did we go for forcing JSON at some point from the REST client?